server/src/main/java/org/apache/iotdb/db/qp/strategy/PhysicalGenerator.java

```diff
@@ -176,6 +176,13 @@ public PhysicalPlan transformToPhysicalPlan(Operator operator) throws QueryProce
               "For Insert command, cannot specified more than one seriesPath: " + paths);
         }
```

Comment:
This check could also be removed. I do not know how to trigger this. The sql parser should have already checked.